internal/pkg/util/interactive/interactive.go

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) 2018-2019, Sylabs Inc. All rights reserved.
```

Comment:
This looks like a new package, so the copyright should be `2019`

```suggestion
// Copyright (c) 2019, Sylabs Inc. All rights reserved.
```